pom.xml

```diff
@@ -498,7 +498,7 @@
     <dependency>
       <groupId>com.hms_networks.americas.sc</groupId>
       <artifactId>extensions</artifactId>
-      <version>1.13.8</version>
+      <version>1.14.1</version>
```

Comment:
Should this PR include adding the reboot watchdog since it was added in this gap?

Reply:
It seems relatively easy to add, so I can quickly put in this PR. 

What is required on the application's part?

Is it just `enableRebootWatchdog()` during connector startup, then call `serviceRebootWatchdog()` each iteration of the main method?

Reply:
You only NEED to call serviceRebootWatchdog() once. Its really up to the application to determine when its "stable". It really shouldn't be dependent on getting an mqtt connection as a disconnected device does not indicate a problem. In what ive tested with AWS im calling it after setting up MQTT and i have been thinking about how to improve that since like i said it shouldnt depend on the mqtt connection (or a wan ip).  Sorry i dont have a clear answer at this point in time. If we do settle on a standard that the watchdog is serviced only after some external connection is setup i would say we should increase the number of acceptable reboots. 

Another option ive been thinking about is adding an optional timer so that you could call serviceRebootWatchdog() at any point but only after say (5 minutes)  does it actually service. That would allow you to service it even if you are stuck waiting for a Wan IP or MQTT connection. I think this is probably the way to go the more i think about it.

I would suggest that calling enableRebootWatchdog() is pretty much the first line of the application.

Reply:
Ok, I'll add enableRebootWatchdog to start of application, then in the main method, check if the watchdog was initialized, and if was and hasn't been serviced, will service it.